odoo/addons/base/models/ir_sequence.py

```diff
@@ -97,6 +98,25 @@ class IrSequence(models.Model):
     _order = 'name'
     _allow_sudo_commands = False
 
+    _BASE_LEGENDS = {
+        'year', 'y', 'month', 'day',
+        'doy', 'woy', 'weekday',
+        'h24', 'h12', 'min', 'sec',
+    }
+
+    _ALLOWED_LEGENDS = {
+        f'%({key})s' for key in _BASE_LEGENDS
+    }.union({
+        f'%(range_{key})s' for key in _BASE_LEGENDS
+    })
```

Comment:
```suggestion
   ALLOWED_LEGENDS = set(
    f'%({prefix}{key})s'
    for prefix in ('', 'range_', 'current_')
    for key in BASE_LEGENDS
)
```
- Forgot adding legends starts with `current_` 
- More clear